src/components/Dashboard/EditAccount.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { Link, NavLink } from 'react-router-dom'
+import SignOut from '../../components/Auth/SignOut'
+import { connect } from 'react-redux'
+
+import axiosWithAuth from '../../Helpers/axiosWithAuth'
+
+import './Dashboard.scss'
+
+function EditAccount({ user }) {
+	const avatar =
+		user.avatar || 'http://s3.amazonaws.com/37assets/svn/765-default-avatar.png'
+
+	const handleSubmit = e => {
+		// e.preventDefault()
```

Comment:
Be sure to uncomment this code.

Keep in mind, the Redux state must be updated to display the user's new `username` or `email`.